File: zig/bismi_allah.zig
//بسم الله الرحمن الرحيم
//la ilaha illa Allah mohammed rassoul Allah

const std = @import("std");
const print = @import("std").debug.print;

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    try stdout.print("بسم الله الرحمن الرحيم\n{s}!\n", .{"la ilaha illa Allah mohammed rassoul Allah"});
    std.debug.print("in the name of Allah\n", .{});
    print("la ilaha illa Allah\n", .{});
}

File: langs/lua/table/bismi_allah.lua
--in the name of Allah
--la ilaha illa Allah mohammed rassoul Allah

print("[lua] in the name of Allah")
print("[lua] la ilaha illa Allah mohammed rassoul Allah")

bismi_allah_table = {}
bismi_allah_table[1] = 12

bismi_allah_metatable = {}
bismi_allah_metatable.__index = function(table, key) return key end

setmetatable(bismi_allah_table, bismi_allah_metatable)

print(bismi_allah_table[1])
print(bismi_allah_table["la ilaha illa Allah"])

